www/class/centreonUtils.class.php

```diff
@@ -330,7 +330,9 @@ public static function escapeSecure(
     ) {
         switch ($escapeMethod) {
             case self::ESCAPE_LEGACY_METHOD:
-                return preg_replace("/<script.*?\/script>/s", "", $stringToEscape);
+                // Remove script and input tags by default
+                $result = preg_replace("/<script.*?\/script>/si", "", $stringToEscape);
```

Comment:
All the replacements could be made with a single _preg_replace()_ call, provided with arrays of strings.